### ISSUE TITLE
Feature reqeust: Examples on using halyard in a Qlik Sense Extension 

### ISSUE THREAD

**Issue by kfsjostrand** [qlik-oss/halyard.js#11]
### Description
I've would like to see examples of using Halyard in a Qlik Sense extension and how to add halyard-enigma mixing to the enigma model. 
Creating regular apps and session apps.

**Comment by ClimberExtensions** [comment]
I've got it to work after I posted an issue on enigma.
var qixSchema = require('text!../.././external/qlik/qix/3.2/schema.json');
var enigmaConfig = {
    Promise: $q,
    schema: JSON.parse(qixSchema),
    session: {
        host: $window.location.host,
        secure: $window.location.protocol === 'https:'
    },
    mixins: enigmaMixin
};

**Comment by StefanEnberg** [comment]
@kfsjostrand I agree that we need an example showcasing how to use halyard.js in a browser setting. If you are interested to learn more about enigma mixins check [halyard-enigma-mixin](https://github.com/qlik-oss/halyard.js/blob/master/src/enigma-mixin/halyard-enigma-mixin.js)
